lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3324,9 +3324,24 @@ class SPIRVJointMatrixINTELInst : public SPIRVJointMatrixINTELInstBase {
 _SPIRV_OP(JointMatrixLoad, true, 6, true)
 _SPIRV_OP(JointMatrixStore, false, 5, true)
 _SPIRV_OP(JointMatrixMad, true, 7)
+// TODO: move to SPIRVJointMatrixINTELWorkItemInst
```

Comment:
Why can't we move this right away in this PR? To preserve backward compatibility by not requiring support of `JointMatrixWIInstructions` capability for now?

Reply:
Usage of `OpJointMatrixWorkItemLength` will currently emit just `CapabilityJointMatrixINTEL`. Which is supported in our downstream. Given the fact, that it's SYCL's counterpart is already implemented and used in tests and user code making the SPIR-V instruction be bound with `CapabilityJointMatrixWIInstructionsINTEL` would make these tests/user code not working, since this capability is not yet introduced in our downstream.

With this patch I'm introducing it, and then, when our drivers adopt it - the TODO can be resolved.